### ISSUE TITLE
Enable IPADOS bugged

### ISSUE THREAD

**Issue by theonewhoknocks11** [khanhduytran0/SparseBox#53]
when enabling on iphone 11 (ios 17.6.1) every app i open immediately crashes, yes by “every” i mean every app, this doesnt apply toward system apps like settings and safari though.

**Comment by khanhduytran0** [comment]
Yeah it may be very buggy with certain models. Use Nugget to undo it.
